libseccomp/src/lib.rs

```diff
@@ -632,18 +632,21 @@ impl ScmpFilterContext {
         Ok(())
     }
 
-    /// add_rule adds a single rule for an unconditional or conditional action on a syscall.
+    /// Adds a single rule for an unconditional action on a syscall.
```

Comment:
Could you add the following description?

```
/// If the specified rule needs to be adjusted due to architecture specifics,
/// it will be adjusted without notification. 
```